src/content/glossary/index.md

```diff
@@ -558,6 +675,10 @@ Referring to the Ethereum network, a peer-to-peer network that propagates transa
   Networks
 </DocLink>
 
+### network hashrate {#network-hashrate}
+
+The number of hash calculations miners on the Ethereum network can make per second collectively.
+
```

Comment:
If we're going to have both "hashrate" and "network hashrate", perhaps link to the other here?
```suggestion
### network hashrate {#network-hashrate}

The collective [hashrate](#hashrate) produced by the entire Ethereum mining network.

```